### ISSUE TITLE
Spaces in "Style"

### ISSUE THREAD

**Issue by IvanAntipov** [tkarabela/pysubs2#45]
I have .ass generated by third-party software. (see [file.zip](https://github.com/tkarabela/pysubs2/files/6777361/file.zip))

This file contains spaces between values in "Style" line

```
Style: Default, Arial, 20, &H00FFFFFF, &H00000000, &H00000000, &H00000000, 0, 0, 0, 0, 100, 100, 0, 0, 1, 2, 0, 2, 15, 15, 15, 0
```
this why pysubs2 can't parse the file

```stack
\lib\site-packages\pysubs2\substation.py in <dictcomp>(.0)
    235                 buf = rest.strip().split(",")
    236                 name, raw_fields = buf[0], buf[1:] # splat workaround for Python 2.7
--> 237                 field_dict = {f: string_to_field(f, v) for f, v in zip(STYLE_FIELDS[format_], raw_fields)}
    238                 sty = SSAStyle(**field_dict)
    239                 subs.styles[name] = sty

\lib\site-packages\pysubs2\substation.py in string_to_field(f, v)
    166                     return timestamp_to_ms(TIMESTAMP.match(v).groups())
    167             elif "color" in f:
--> 168                 return rgba_to_color(v)
    169             elif f in {"bold", "underline", "italic", "strikeout"}:
    170                 return v == "-1"

\lib\site-packages\pysubs2\substation.py in rgba_to_color(s)
     72         x = int(s[2:], base=16)
     73     else:
---> 74         x = int(s)
     75     r = x & 0xff
     76     g = (x >> 8) & 0xff

ValueError: invalid literal for int() with base 10: ' &H00FFFFFF'
```

Also line 
```[v4+ Styles]``` starts with small letter 'v' and I  have to specify 'format_' explicitly.

Probably, it will be a good idea to remove spaces from "Style" line and do invariant comparison during guess of file format?

**Comment by tkarabela** [comment]
Thanks for the report, I'll implement it and make a new release :)

**Comment by tkarabela** [comment]
A little late, but I've released new version 1.3.0 of the library which fixes this :)
